README.md

```diff
@@ -1,40 +1,55 @@
+
 ![](images/logicapp.PNG)
-Also there is a GitHub repository that has custom logic apps that any one can deploy here is the [link ](https://github.com/Azure/logicapps)
-## Hunting 
-Hunting is a term in Sentinel were using the search and query tools to proactively hunt for security threats across all data sources, before an alert is triggered. After the threat is deactivated you can make custom detection rules based on the query. Here is list of built-in queries Azure provides. 
+
+Also, there is a [GitHub repository that has custom Logic Apps](https://github.com/Azure/logicapps) that anyone can deploy.
+
+## Hunting
+Hunting in Sentinel where one uses the search and query tools to proactively hunt for security threats across all data sources before an alert is triggered. After the threat is deactivated you can make custom detection rules based on the query. Here is list of built-in queries Azure provides.
+
 ![](images/hunting.PNG)
-## Threat Intelligence 
-Threat Intelligence has become a hot topic in the security community as of late. Threat Intelligence is available knowledge that can mitigate or prevent cyberattacks. Threat Intelligence can help companies understand what attackers are doing to their systems and how companies should to respond to those attacks. This can enhance any companies' ability to detect and prioritize known threats. Azure Sentinel does this by having dedicated Analytics, workbooks, hunting query, notebooks and data connectors for Threat Intelligence.  
 
-Unfortunately, Azure doesn't have any automattic threat intelligence built in however, Azure Sentinel provides some options for companies to acquire threat intelligence. Azure Sentinel gives companies the ability to import threat indicators that they use. 
+## Threat Intelligence
+Threat Intelligence has become a hot topic in the security community as of late. Threat Intelligence is available knowledge that can mitigate or prevent cyberattacks. Threat Intelligence can help companies understand what attackers are doing to their systems and how companies should respond to those attacks. This can enhance any company's ability to detect and prioritize known threats. Azure Sentinel does this by having dedicated Analytics, Workbooks, Hunting Queries, Notebooks, and Data Connectors for Threat Intelligence.
 
-There are multiple ways of streaming threat indicators to Azure Sentinel. You can use one of the Threat Intelligence Platform products that is integrated with Azure, you can connect a TAXII server to Azure Sentinel via a data connector or you can use direct integration with the  Microsoft Graph Security tiInicators API. 
+Unfortunately, Azure Sentinel doesn't have any automattic threat intelligence built-in. However, it does provide some options for companies to acquire threat intelligence. Azure Sentinel gives companies the ability to import threat indicators that they use.
```

Comment:
"automattic" = "automatic"

Reply:
maybe change "acquire threat intelligence" to "import their own threat intelligence sources"